e2e/tests/nightly/gke-apps/cromwell.spec.ts

```diff
@@ -134,14 +136,40 @@ describe('Cromwell GKE App', () => {
     expect(submitJob.includes('Submitting job to server'));
     expect(submitJob.includes('"status": "Submitted"'));
 
-    // TODO: Add more steps for the submitted cromwell jobs like confirm its running etc
-    // In the meantime we will go ahead, delete cromwell and start cleanup
-    const isDeleted = await appsPanel.deleteCromwellGkeApp();
+    const jobId = submitJob.split('    "id": "')[1].split('",')[0];
+    console.log(jobId);
```

Comment:
Could you add a little text context here?  Maybe "submitted {jobId} to cromwell" ?

Reply:
Added console logs example also the stages/status  for all Jobs

Reply:
The comment is very nice, thanks!

What I meant was: when this runs, we will see a line like this with no context about what it means:
```
console.log - 12345-421453-13234241
```

It would be more helpful to see something like this instead:
```
console.log - Cromwell job submitted.  ID = 12345-421453-13234241
```